cmd/ctr/snapshot.go

```diff
@@ -200,7 +201,7 @@ var removeSnapshotCommand = cli.Command{
 
 var prepareSnapshotCommand = cli.Command{
 	Name:      "prepare",
-	Usage:     "prepare gets mount commands for digest",
+	Usage:     "prepare activates snapshot for digest",
```

Comment:
This language doesn't exactly make sense. It should be `prepare a snapshot from a tar file by digest`.